MachObfuscatorTests/SymbolManglingTests/RealWordsExportTrieMangler_Tests.swift

```diff
@@ -50,27 +45,35 @@ class RealWordsExportTrieMangler_Tests: XCTestCase {
         XCTAssertEqual(obfuscatedTrie.children[2].children[0].label, [0, 0])
         XCTAssertEqual(obfuscatedTrie.children[2].children[1].label, [1, 1])
     }
+}
+
+private typealias TrieLevel = (labelLength: UInt, childrenCount: UInt)
```

Comment:
`Int` is preferred in Swift over `UInt` and here you do not require `UInt` (the same for `count` in `random`)

Reply:
Um, I had to read [that](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html#ID320). You are right, thanks :)